pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1333,6 +1333,15 @@
     <StablePackage Include="System.IO.Compression">
       <Version>4.1.1</Version>
     </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.Platforms">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http">
+      <Version>4.1.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http.WinHttpHandler">
```

Comment:
Folks can still reference this directly.  It wouldn't make sense to update HttpClient and have fixes but then when you pass in a WinHttpHandler to not get the same fixes.